CHANGELOG.md

```diff
@@ -43,6 +43,8 @@ and this project adheres to
   - [#1866](https://github.com/iovisor/bpftrace/pull/1866)
 - Fix memory leak in clang parser
   - [#1878](https://github.com/iovisor/bpftrace/pull/1878)
+- Fix memory leaks in struct types
```

Comment:
wrong section due to 0.13 release, needs rebase

Reply:
rebased